### DISCUSSION TITLE
What issues can I expect upgrading from TeXstudio 2.12.6 to the current 4.2.3 ?

### DISCUSSION THREAD

**Discussion by jb1957** [texstudio-org/texstudio#2302] [General]
My department is still running TeXstudio 2.12.6 due to its stability for our purposes and to avoid the unknowns that might break if we upgraded to the current version 4.2.3.

However, I'm tossing out the question to those who are experienced in installing upgrades in a small academic environment where about 15 people use TeXstudio version 2.12.6. 

Can we expect any major problems in upgrading from 2.12.6 to 4.2.3? Could there be any backward compatibility problems? Any hidden "gotchas" to look out for? 

Thanks.

**Comment by sunderme** [comment]
any particular reasons why you don't just try it out ?
We are talking about tex i.e. plain text source files, so unless you use really old text coding (inputenc), there is not much that can go south.
